sdk/cosmosdb/cosmos/tsconfig.json

```diff
@@ -1,24 +1,18 @@
 {
+  "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "declaration": true,
-    "declarationMap": true,
     "module": "esnext",
-    "moduleResolution": "node",
-    "importHelpers": true,
+    "strict": false,
     "noImplicitAny": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
+    "noImplicitReturns": false,
+    "noFallthroughCasesInSwitch": false,
```

Comment:
Hello!

The rest of the PR is specifically related to Identity. These changes don't seem to be related with Identity, nor with the issues you've linked this PR to. Perhaps I'm missing something! Would you mind helping me see why this is necessary?

Have a good day!

Reply:
@sadasant Actually I created a branch in my fork repo based on the [extend-config](https://github.com/ramya-rao-a/azure-sdk-for-js/tree/extend-config) branch PR #11424 by @ramya-rao-a which is not yet merged into master. 
So that is why this PR showing file change for cosmos tsconfig.json

Reply:
@mohsin-mehmood, If this PR was made to my branch as the base branch, then you would not have seen those diffs.

Regardless, #11424 is merged. Please merge latest changes from master to your branch you so that this PR can show only the changes you made

Reply:
After merging master branch into my branch cosmos project tsconfig.json file is not part of this PR

Reply:
Thank you!